eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -73,7 +75,16 @@ jobs:
         value: ${{ parameters.Location }}
 
     strategy:
-      matrix: ${{ parameters.Matrix }}
+      matrix: 
+        ${{ insert }}: ${{ parameters.Matrix }}
+        ${{ if eq(parameters.TestCanary, 'true') }}:          
+          Linux - Java 11 (AzureCloudCanary):            
+            OSVmImage: 'ubuntu-18.04'
+            JavaTestVersion: '1.11'
+            DisplayName: 'Linux - Java 11'          
+            SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)            
```

Comment:
As an aside for Canary should we be using the "-preview" or the "-dogfood" subscription

Reply:
@danieljurek suggested '-preview' for special cases like this since we shouldn't disturb the main (normal) subscription with new/more permissions. We haven't talked about '-dogfood' subscription, I'll leave it to you to decide.

Reply:
If this is only about enabling preview feature flags then we should be OK with using "-preview".